src/main/java/com/ericsson/ei/jsonmerge/MergePrepare.java

```diff
@@ -135,8 +135,10 @@ public String getMergePathFromArrayMergeRules(String originObject, String mergeR
         log.debug(" originObject is : " + originObject);
         try {
             JSONArray ruleJSONArray = new JSONArray(mergeRule);
-            String firstRule = ruleJSONArray.getString(0);
-            String secondRule = ruleJSONArray.getString(1);
+            String firstRule = "";
+            String secondRule = "";            
```

Comment:
What is firstRule and secondRule? Make it more readable by choosing better variable names.